pkg/network/tracer/process_cache_test.go

```diff
@@ -339,3 +340,24 @@ func TestProcessCacheGet(t *testing.T) {
 	}
 
 }
+
+func BenchmarkProcessCacheMem(b *testing.B) {
+	pc, _ := newProcessCache(b.N, nil)
```

Comment:
I think you want to use a constant size here -- why did you use b.N?

Reply:
@hmahmood suggested it to see the cache increase when b.N increased. We tried a few numbers. Do you have a suggestion as to what the size might be?

Reply:
> it to see the cache increase when b.N increased

I think I might be missing something, but is that what we want?

I would suggest any constant, it doesn't really matter what. 

In a benchmark, the thing inside the `for i := 0; i < b.N; i++` should have a constant cost no matter what `b.N` is.

Reply:
Its an LRU, so if `b.N > size`, we'll see evicts. I just wanted to eliminate that possibility.